### ISSUE TITLE
Weekly Report: Week of 2024-10-14

### ISSUE THREAD

**Issue by rui-zhou-ops** [rui-zhou-ops/zhoubao#1]
#1.  **paper**
Deep learning for Inertial postioning:A survey

#2.**summary**
This survey introduces the application of deep learning in inertial positioning about the sensor level, algorithm level and application level.
(1)At the sensor level, deep learning is employed as a calibration method for inertial sensors. It effectively models error sources in inertial measurements and implicitly eliminates corrupted measurement errors and noise.
(2)At the algorithm level, deep neural networks are constructed to partially or completely replace modules in traditional inertial navigation mechanisms that enhance and correct IMU integration in general.
(3)At the application level, main introduce  specific use cases where deep learning methods can be applied to inertial positioning for pedestrians, vehicles, drones, and
robots.
(4)This survey also introduces the advantages and challenges of deep learning in inertial positioning.

#3.**plan**
To summarize the survey, fully understand it and make a ppt presentation.

#4.**problem**
When reading the survey, some concepts are vague and some approaches are not summary.
